backend/src/test/java/com/redhat/cloud/notifications/routers/UserConfigResourceTest.java

```diff
@@ -483,16 +515,30 @@ void testSettingsByEventType() {
         featureFlipper.setInstantEmailsEnabled(true);
 
         // Daily and Instant to false
-        updateAndCheckUserPreference(path, identityHeader, bundle, application, eventType, false, false);
+        updateAndCheckUserPreference(path, identityHeader, bundle, application, eventType, false, false, true);
 
         // Daily to true
-        updateAndCheckUserPreference(path, identityHeader, bundle, application, eventType, true, false);
+        updateAndCheckUserPreference(path, identityHeader, bundle, application, eventType, true, false, true);
 
         // Instant to true
-        updateAndCheckUserPreference(path, identityHeader, bundle, application, eventType, false, true);
+        updateAndCheckUserPreference(path, identityHeader, bundle, application, eventType, false, true, true);
 
         // Both to true
-        updateAndCheckUserPreference(path, identityHeader, bundle, application, eventType, true, true);
+        updateAndCheckUserPreference(path, identityHeader, bundle, application, eventType, true, true, true);
+
+        if (featureFlipper.isDrawerEnabled()) {
+            // Daily and Instant to false
+            updateAndCheckUserPreference(path, identityHeader, bundle, application, eventType, false, false, false);
+
+            // Daily to true
+            updateAndCheckUserPreference(path, identityHeader, bundle, application, eventType, true, false, false);
+
+            // Instant to true
+            updateAndCheckUserPreference(path, identityHeader, bundle, application, eventType, false, true, false);
+
+            // Both to true
+            updateAndCheckUserPreference(path, identityHeader, bundle, application, eventType, true, true, false);
+        }
```

Comment:
Why is the last arg `false` when the drawer is enabled and `true` in the first 4 `updateAndCheck...`? Shouldn't it be the opposite?

Reply:
Actually a part of this test was missing.